### ISSUE TITLE
Create TESTING.md in Centaur

### ISSUE THREAD

**Issue by kcibul** [broadinstitute/cromwell#1634]
As a user of Cromwell, I would like to know what is being tested via Centaur.  This is important so that I don't overtest things that are being tested and also don't stress out about things because I don't know if they are being tested.

I would like a document that lists each test (WDL) being run and a sentence about what it is doing

**Comment by katevoss** [comment]
@kcibul when or why would a Cromwell user want to know what Centaur is testing? Is this a Cromwell user who is going direct to Cromwell, or via some other portal?

**Comment by geoffjentry** [comment]
@katevoss Good examples would be GOTC or Firecloud, where they expend effort to test things before incorporating a new version into their system. It often happens that we discover that and tell them "we already test X", but they have no way of knowing.

**Comment by kcibul** [comment]
yep -- that's where this came from, basically documenting our tests (in a very lightweight manner).  Might not also be bad for Cromwell internally rather than trying to guess what a certain WDL tests although the names are quite clever ;)

**Comment by katevoss** [comment]
As a **workbench QA**, I want **to know what is being tested**, so that I **don't under or over test for each release**.
@geoffjentry 
- Effort: **TBD**
- Risk: **TBD**
- Business value: **TBD**

**Comment by geoffjentry** [comment]
risk is negligible 
effort is probably higher than it should be but still shouldn't be too much
